requirements.txt

```diff
@@ -1,85 +1,567 @@
+protobuf==6.30.2
+    # via
+    #   onnx
+    #   onnxruntime
+    #   tensorboard
+    #   wandb
+psutil==7.0.0
+    # via
+    #   -r spinal_requirements.in
+    #   totalspineseg
+    #   wandb
+pybids==0.15.5
+    # via ivadomed
+pydantic==2.11.4
+    # via wandb
+pydantic-core==2.33.2
+    # via pydantic
+pydicom==2.4.4
+    # via dicom2nifti
+pygments==2.19.1
+    # via rich
+pyparsing==3.2.3
+    # via matplotlib
 pyqt5==5.12.3
-# PyQt5-sip should be updated when PyQt5 is updated: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4354
-pyqt5-sip<12.13.0
-# pystrum==0.2 is incompatible with newer versions of numpy (installed via voxelmorph -> neurite -> pystrum)
-# See also: https://github.com/adalca/pystrum/issues/9
-pystrum>=0.3
-pytest
-pytest-cov
-requests
-requirements-parser
-scipy
-scikit-image
-scikit-learn
-totalspineseg
-xlwt
-# - torch<2.6: In January 2025, PyTorch changed the default of `torch.load()` to `weights_only=True`, breaking the
-#              loading of some of our models. We can set `weights_only=False` to retain the old behavior, however
-#              there are additional calls to `torch.load()` within our dependencies as well, meaning that we need to
-#              wait until they update themselves (or become compatible with `weights_only=True`). Until then, we can
-#              pin torch to a version prior to this breaking change.
-# - torch<2.3: PyTorch 2.3.0 removed support for Intel Macs. If we leave this unpinned, then version 2.2.2 will be
-#              installed on Intel Macs, while a newer version (e.g. 2.5.1) will be installed on all other platforms.
-#              To keep the versions in sync between platforms, we pin torch to `<2.3`. See:
-#              -> https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4781
-# - NOTE: nnunetv2 currently requries `torch>=2.1.2`, meaning that the above restriction results in a range of
-#         `2.1.2 <= torch < 2.3`, i.e. most likely version `2.2.2` will be installed.
-torch<2.3 # append_to_freeze
-tqdm
-transforms3d
-urllib3
-pytest_console_scripts
-pyyaml
-voxelmorph
-wquantiles
-# XlxsWriter is needed to write `.xlsx` files, since pandas removed support for the `xlwt` engine
-# Source: https://github.com/pandas-dev/pandas/pull/49296
-xlsxwriter
+    # via -r spinal_requirements.in
+pyqt5-sip==12.12.2
+    # via
+    #   -r spinal_requirements.in
+    #   pyqt5
+pyreadline3==3.5.4
+    # via humanfriendly
+pystrum==0.4
+    # via
+    #   -r spinal_requirements.in
+    #   neurite
+pytest==8.3.5
+    # via
+    #   -r spinal_requirements.in
+    #   pytest-console-scripts
+    #   pytest-cov
+pytest-console-scripts==1.4.1
+    # via -r spinal_requirements.in
+pytest-cov==6.1.1
+    # via -r spinal_requirements.in
+python-dateutil==2.9.0.post0
+    # via
+    #   matplotlib
+    #   pandas
+python-gdcm==3.0.24.1
+    # via dicom2nifti
+pytz==2025.2
+    # via pandas
+pywin32==310
+    # via portalocker
```

Comment:
It looks like freezing the requirements on Windows is causing some errors on the non-Windows OSs due to deps like `pywin32`.